openstack/sfs_turbo/v1/shares/requests.go

```diff
@@ -206,12 +214,20 @@ type ExtendOpts struct {
 	BssParam *BssParamExtend `json:"bss_param,omitempty"`
 }
 
+type ChangeNameOpts struct {
+	Name string `json:"name" required:"true"`
+}
+
 // ToShareExpandMap assembles a request body based on the contents of a
 // ExpandOpts.
 func (opts ExpandOpts) ToShareExpandMap() (map[string]interface{}, error) {
 	return golangsdk.BuildRequestBody(opts, "")
 }
 
+func (opts UpdateNameOpts) ToShareUpdateNameMap() (map[string]interface{}, error) {
+	return golangsdk.BuildRequestBody(opts, "")
+}
```

Comment:
There is a simple way to define the opts

```suggestion
type UpdateNameOpts struct {
	Name string `json:"name" required:"true"`
}

func (opts UpdateNameOpts) ToShareUpdateNameMap() (map[string]interface{}, error) {
	return golangsdk.BuildRequestBody(opts, "change_name")
}
```